src/main/java/org/gridsuite/merge/orchestrator/server/MergeEventService.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```

Comment:
I think we should keep 2020 as the file was created in 2020

Reply:
Done